src/main/java/dev/jbang/cli/Run.java

```diff
@@ -75,7 +76,10 @@ public Integer doCall() throws IOException {
 			if (!literalScript.get().isEmpty()) {
 				script = literalScript.get();
 				if (scriptOrFile != null) {
-					userParams.add(0, scriptOrFile);
+					List<String> args = new ArrayList<>();
+					args.add(scriptOrFile);
+					args.addAll(ctx.getArguments());
```

Comment:
is Arguments and useParams the same ? I thought userParam had to come before the arguments?

Reply:
I took a closer look and yes RunContext.arguments and Run.userParams are the same. In the old code the extra script argument would get added to the start of Run.userParams and then that would be copied to RunCOntext.arguments. In the new code RunContext.arguments has already been initialized with Run.userParams and this code then adds an extra argument to the start. Again, the behaviour should be exactly the same (no tests failed at least, but we know that isn't always a good indication)